docs/modules/ROOT/pages/logging/tutorial-lokistack.adoc

```diff
@@ -0,0 +1,109 @@
+= Tutorial: Access application logs with Lokistack
+
+This tutorial explains how to view the logs of an application on Openshift.
+
+
+== Requirements
+
+To follow this guide, please make sure that you have the following tools installed:
+
+`oc`:: You can download the OpenShift command directly from {product}, selecting the help menu (marked as a question mark) and selecting the "Command line tools" entry
+
+
+== Step 1: Deploy Fortune application
+
+. Create a new namespace called `vshn-demo-fortune`
++
+[source,shell]
+--
+oc create ns vshn-demo-fortune
+--
+
+. Create Fortune application manifest
++
+.Manifest for Fortune application
+[source,yaml]
+--
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: fortune-cookie
+  name: fortune-cookie
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: fortune-cookie
+  template:
+    metadata:
+      labels:
+        app: fortune-cookie
+    spec:
+      containers:
+      - image: registry.gitlab.com/vshn/applications/fortune-go:latest
+        imagePullPolicy: IfNotPresent
+        name: fortune-cookie
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: fortune-cookie
+  name: fortune-cookie
+spec:
+  ports:
+  - port: 8080
+    protocol: TCP
+    targetPort: 8080
+    name: web
+  selector:
+    app: fortune-cookie
+  type: ClusterIP
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-production
+  name: fortune-cookie
+spec:
+  rules:
+  - host: <INSERT_URL_FOR_FORTUNE_APP>
+    http:
+      paths:
+      - pathType: Prefix
+        path: /
+        backend:
+          service:
+            name: fortune-cookie
+            port:
+              number: 8080
+  tls:
+  - hosts:
+    - <INSERT_URL_FOR_FORTUNE_APP>
+    secretName: fortune-cookie-tls
```

Comment:
I'd consider leaving out the ingress, and instead using `oc expose svc/fortune-cookie` instead for the tutorial, something like

```
oc -n vshn-demo-fortune expose svc/fortune-go
oc -n vshn-demo-fortune patch route fortune-go -p '{"spec":{"tls":{"termination":"edge","insecureEdgeTerminationPolicy":"Redirect"}}}'

```